### DISCUSSION TITLE
Is there a possibility to change the size of the scrollbar of the sidebar?

### DISCUSSION THREAD

**Discussion by ralf3u** [geany/geany#3953] [Q&A]
To change the size of the scrollbar of the editor, one can add for example this code in the document geany.css that is located in .geany:
```
/* size of the scrollbar */
scrollbar slider {
    min-width: 10px;
    border-right: 10px solid transparent;
}
```

But the code has no influence on the scrollbar of the sidebar.
How can I change the size of the scrollbar of the sidebar?
Thank you in advance for an answer.

**Comment by elextr** [comment]
WFM.  But only when the sidebar scrollbar is visible under the insane hiding behaviour of GTK (sidebar has a normal GTK scrollbar).  The editor sidebars are controlled by Scintilla and follow Scintilla scrollbar behaviour which is only slightly eccentric (ie not as insane as GTK).  I guess its the least common behaviour Scintilla can use in common between Qt, GTK, Windows, and Macos.

Perhaps GTK CSS has the ability to control the hiding behaviour, but I don't know it.  Perhaps ask on a GTK CSS forum.

**Comment by ralf3u** [comment]
>WFM

Sorry to ask, but WFM means "works for me", right? So, if I understand it right, if you change the size of the scrollbar by CSS (see code in the report), then it does not only change the size of the scrollbar of the editor but also the size of the scrollbar of the sidebar.
How is it possible, that this works for you, but not for me?
I use Geany 2.0 with this info: GTK 3.24.41, GLib 2.80.0.

**Comment by elextr** [comment]
> Sorry to ask, but WFM means "works for me", right?

Yes, sorry.

>  (see code in the report),

Used copy and paste of that CSS but with 30px just to make sure its obvious.

> I use Geany 2.0 with this info: GTK 3.24.41, GLib 2.80.0.

Well, my version is Geany Git version, and slightly older GTK/Glib.  Remember the sidebar is standard GTK only, nothing special so it should work for your newer versions unless GTK broke.  

Just to check, you did close and restart Geany each change of the CSS.

**Comment by ralf3u** [comment]
>Just to check, you did close and restart Geany each change of the CSS.

After each CSS change, I close and restart Geany. But neither `min-width` nor `border-right` has an influence on the size of the scrollbar of the sidebar.

What should I do?

**Comment by ralf3u** [comment]
>WFM

I just recognized that it affects indeed the size of the scrollbar of the sidebar for the tab `Documents` and for the tab `Files` (plugin `File Browser`), but not the tab `Preview` (plugin `Preview`).  So, it seems to be that the plugin `Preview` is the problem. I will contact the maintainer of the plugin `Preview`.

Thank you for your help.
